cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
I'm not saying that you did anything wrong.  You added some arguments to `republish_collection`, you should make sure those changes work, which you did.